src/utils/email.js

```diff
@@ -43,7 +43,7 @@ async function initEmail (smtpUrl = AppConstants.SMTP_URL) {
  * @param {string} recipient
  * @param {string} subject
  * @param {string} html
- * @returns <Promise>
+ * @returns {Promise} <Promise>
```

Comment:
Hm, I wonder why `eslint-plugin-jsdoc` didn't complain about this before.

Reply:
It did complain and showed a warning for me. That’s what prompted me to fix it. Would you prefer being more restrictive and throw an error if this rule is not satisfied?

Reply:
Oh no that's fine thanks!